.github/workflows/coverage.yaml

```diff
@@ -1,18 +1,17 @@
 # Performs test coverage of project's libraries using cargo-tarpaulin and generates results using codecov.io.
 # The following flags are set inside `tarpaulin.toml`:
 # `features = "..."`: Includes the code with the listed features. The following features result in a
-#     tarpaulin error and are NOT included: derive, alloc, arbitrary-derive, attributes, and
```

Comment:
Please clerify what have changed in this paragraph.
Also, I dont think you modified the `timeout` here to 1200s, which was changed here https://github.com/stratum-mining/stratum/pull/1197/commits/105acda841d67178846402129a96cd088678b471#diff-8965ddb70c4bbfcd7467189aba2a7d41675f85a1f5ff23119bea37504b96f351R4

Reply:
Just formatting changed... that timeout is now restored to 120s